runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/ThreeStepCrypticClue.java

```diff
@@ -177,10 +180,14 @@ public static ThreeStepCrypticClue forText(String plainText, String text)
 						// chop off the length of the clue text plus a space
 						threeStepText.delete(0, clue.getText().length() + 1);
 					}
-
+					clueMatched = true;
 					break;
 				}
 			}
+			if (!clueMatched)
```

Comment:
you can probably just do this in steps.empty or even from ClueScrollPlugin, where resetClue is called in the widget check, that would also solve the other cryptic clue issue